packages/block-editor/src/hooks/layout.js

```diff
@@ -153,17 +152,22 @@ function LayoutPanel( { setAttributes, attributes, name: blockName } ) {
 		return null;
 	}
 
+	// Layout block support comes from the block's block.json.
 	const layoutBlockSupport = getBlockSupport(
 		blockName,
 		layoutBlockSupportKey,
 		{}
 	);
+	const blockSupportAndThemeSettings = {
+		...layoutSettings,
+		...layoutBlockSupport,
+	};
```

Comment:
I'm undecided about the order of these. On one hand, it would make sense for it to be possible to override block default settings at a theme level. On the other, enabling layout controls on some blocks might result in a suboptimal experience (e.g. Gallery or Columns).

Reply:
My understanding was that we'd want `theme.json` to be able to disable something set in `block.json` (i.e. hide controls) but never to enable something that the block itself hasn't opted into.

Reply:
Then the order is correct, assuming controls aren't explicitly set to `true` in block.json. Which they shouldn't be, because all controls display by default.